### ISSUE TITLE
Install on clean SD-Card failed after Reboot

### ISSUE THREAD

**Issue by m-sja** [FABtotum/fabui-colibri#201]
After formatting the SD-Card and unzipping the files to it, i run the initial Setup.
(Language, Account, etc ...)

During Install i had HDMI connected to the RPi1a.
I got the following error messages:
![firstboot_1](https://user-images.githubusercontent.com/28825834/30288502-41afe070-9729-11e7-96d9-f17eecdb7227.png)
![firstboot_2](https://user-images.githubusercontent.com/28825834/30288510-48f29184-9729-11e7-95fd-a757bfa0134b.png)

In the UI setup process there is an error (i tried Englisch and German).

On step 4 the dialog refers to the "Install"-Button, but there is none.
The only button wich works is "Next", but the cursor is changing to a "Forbidden" Symbol.
But it works ?confused?
![setup_4](https://user-images.githubusercontent.com/28825834/30287748-853fa68e-9726-11e7-9c01-6a463f801782.png)

The Setup process finishes and the Printer is usable until reboot.
Then i got these errors leading to kernel panic:
![after_reboot](https://user-images.githubusercontent.com/28825834/30288665-e22c3bac-9729-11e7-881d-eb14161aca01.png)

I tried two different Pi1a with two SD-Cards, Micro-SD-Card and one Pi1b. - No Luck
I tried all the available fabui-colibri zip-files from [here](http://update.fabtotum.com/admin/list.php) - No Luck
It seems that fabui-colibri is running so well and stable, that nobody gets to the point where i am now ;-)

Thanks for Help.

**Comment by m-sja** [comment]
Btw.
This is the storage-information page after the initial Setup.
It seems there are major issues regarding partitions:
![storage_information](https://user-images.githubusercontent.com/28825834/30291727-0658b41a-9734-11e7-9bdd-004ef78fa755.png)

**Comment by infinity0n3** [comment]
The partitions were not created at all. Did you follow the instructions for formatting the sdcard?
[SDCard Formatting](https://github.com/FABtotum/fabui-colibri/wiki/SD-card-formatting)

What OS do you use to format the sdcard?
What is the size of your sdcards?

**Comment by daniel-fabteam** [comment]
The partitions were not created at all. Did you follow the instructions for formatting the sdcard?
[SDCard Formatting](https://github.com/FABtotum/fabui-colibri/wiki/SD-card-formatting)

What OS do you use to format the sdcard?
What is the size and type (SD, SDHC, SDXC) of your sdcards?

**Comment by m-sja** [comment]
Hey Daniel, thanks for the hint.
I'm using ubuntu mate 16.4 lts.
I formatted the SD-Cards as i always did (deleting all partitions, creating a new W95 FAT32 c, boot flag).
But i was not aware, that there is a "Special" way to do it. It always worked except for the issues above.

I tried the gui (Linux) method step by step and got an error while formatting the new created Partition.
`Error synchronizing after initial wipe: Timed out waiting for object (udisks-error-quark, 0)`
The solution was to change the card-reader! The internal reader of my thinkpad seems to cause some issues. A generic cheap reader solved the issue.

One 16gb Samsung SDHC is working now. :+1: 
The original "FABtotum" 8gb (wich came with the Printer) formats and installs fine, but is still giving errors mounting "bigtemp" from /dev/mmcblk0p5. The other partitions are fine. I assume the Card is dead somehow.
With my 8gb SanDisk Ultra micro-sdhc the RPi refuse to start if i format your way.

It's kinda weird, but at least i have one working card.
